.github/workflows/static-analysis.yml

```
@@ -19,7 +19,7 @@ jobs:
                 include:
                     - { php-version: "8.1", dependencies: "lowest", require_admin_bundle: false }
                     - { php-version: "8.2", dependencies: "highest", require_admin_bundle: true }
-                    - { php-version: "8.3", dependencies: "highest", pimcore_version: "11.x-dev as 11.0.0", require_admin_bundle: true }
+                    - { php-version: "8.3", dependencies: "highest", pimcore_version: "11.x-dev as 11.9.99", require_admin_bundle: true }
```

```suggestion
                    - { php-version: "8.3", dependencies: "highest", pimcore_version: "11.x-dev", require_admin_bundle: true }
```
as it would be managed by branch alias in the Core
https://github.com/pimcore/pimcore/blob/ee9bc5b162e68c25428963c693dfe980884d5ceb/composer.json#L203-L204